src/Scoper.php

```diff
@@ -33,12 +30,14 @@ public function __construct(Parser $parser)
     }
 
     /**
-     * @param $content
-     * @param $prefix
+     * @param string $content Content of the file to scope
+     * @param string $prefix Prefix to apply to the file
+     *
+     * @throws ParsingException
      *
-     * @return string
+     * @return string Content of the file with the prefix applied
      */
-    public function addNamespacePrefix($content, $prefix)
+    public function addNamespacePrefix(string $content, string $prefix): string
```

Comment:
I'm assuming not declaring strict at top of file is intended (here and a few other files)?